examples/jsm/loaders/ColladaLoader.js

```diff
@@ -3926,11 +3942,11 @@ ColladaLoader.prototype = {
 		var textureLoader = new TextureLoader( this.manager );
 		textureLoader.setPath( this.resourcePath || path ).setCrossOrigin( this.crossOrigin );
 
-		var tgaLoader;
+		var tgaLoader = this.tgaLoader;
 
-		if ( TGALoader ) {
+		if ( tgaLoader ) {
 
-			tgaLoader = new TGALoader( this.manager );
+			tgaLoader.manager = this.manager;
```

Comment:
Need to regenerate the JSM version I assume?

Reply:
Done 👍